submissions/Serge044/HTML-CSS-Popup/index.html

```diff
@@ -0,0 +1,169 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Serge's popup project</title>
+    <link rel="shortcut icon" href="images/favicon.png">
+    <link rel="stylesheet" href="style.css">
+</head>
+
+<body>
+    <div class="wrapper">
+        <header class="header">
+            <nav class="nav__menu">
+                <ul class="ul__items">
+                    <li>
+                        <a href="#Mail" class="main__nav-buttons mail"><span
+                                class="main__nav-buttons__text">Mail</span></a>
+                    </li>
+                    <li>
+                        <a href="#Images" class="main__nav-buttons images"><span
+                                class="main__nav-buttons__text">Images</span></a>
+                    </li>
+                    <li>
+                        <input class="popup_btn" type="checkbox">
+                        <div class="popup__menu">
+                            <ul class="popup__menu__main">
+                                <li class="popup__menu__item">
+                                    <a href="#Calendar">
+                                        <img src="images/kottans-calendar.png" alt="Calendar" class="popup__icons">
```

Comment:
For icons it's better set an empty alt attribute, read more [here](https://accessibility.psu.edu/images/imageshtml/#empty)